### PR TITLE
Remove mention of the Talons project from the tutorial, update FAQ

### PR DESCRIPTION
**Pull request by kgriffs**
The Talons project has not become a general repository for
community add-ons, as originally envisioned. Remove from the
the tutorial to avoid confusion (based on community feedback.)

Also update the FAQ as follows:

    * Remove mention of talons to allow for diversity in
      the Falcon ecosystem
    * Remove mention of specific "choices" that Falcon lets
      you make regarding complementary libraries, since in
      each case there are actually many more choices than
      given, and the list only seems to belabor the point.

### CODE REVIEW

File: doc/community/faq.rst [hunk 1]
```diff
@@ -18,39 +18,21 @@ simply wrap your api instance with a middleware app. For example:
 
 See also the `WSGI middleware example <http://legacy.python.org/dev/peps/pep-3333/#middleware-components-that-play-both-sides>`_ given in PEP-3333.
 
-
 Why doesn't Falcon include X?
 -----------------------------
-The Python ecosytem offers a bunch of great libraries that you
```
Comment (fxfitz) [inline doc/community/faq.rst]: Nit: Maybe rephrase to 'Why doesn't Falcon include <X>?'

Comment (fxfitz) [inline doc/community/faq.rst]: Oh, well, looks like markdown took out my "< X >"

Comment (kgriffs) [inline doc/community/faq.rst]: I rewrote this. Let me know what you think.

**Comment by fxfitz** [comment]
LGTM
